transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/LogMessages.kt

```diff
@@ -1,3 +1,97 @@
 package com.genesys.cloud.messenger.transport.util.logs
 
-object LogMessages
+object LogMessages {
+    const val UNHANDLED_MESSAGE = "Unhandled message received from Shyrka: "
+    const val REFRESH_AUTH_TOKEN_SUCCESS = "refreshAuthToken success."
+    fun PRESIGNING_ATTACHMENT(attachment: String) = "Presigning attachment: $attachment"
+    fun UPLOADING_ATTACHMENT(attachment: String) = "Uploading attachment: $attachment"
+    fun ATTACHMENT_UPLOADED(attachment: String) = "Attachment uploaded: $attachment"
+    fun DETACHING_ATTACHMENT(attachmentId: String) = "Detaching attachment: $attachmentId"
+    fun ATTACHMENT_DETACHED(attachmentId: String) = "Attachment detached: $attachmentId"
+    fun SENDING_ATTACHMENT(attachmentId: String) = "Sending attachment: $attachmentId"
+    fun ATTACHMENTS_SENT(attachments: String) = "Attachments sent: $attachments"
+    fun ADD_CUSTOM_STATE(customAttributes: String, state: String) =
+        "add: $customAttributes | state = $state"
+    const val ON_SENDING = "onSending"
+    fun ON_SENT_STATE(state: String) = "onSent. state = $state"
+    const val ON_ERROR = "onError"
+    const val ON_MESSAGE_ERROR = "onMessageError()"
+    const val ON_SESSION_CLOSED = "onSessionClosed"
+    fun MESSAGE_PREPARED_TO_SEND(message: String) = "Message prepared to send: $message"
+    fun MESSAGE_STATE_UPDATED(message: String) = "Message state updated: $message"
+    fun ATTACHMENT_STATE_UPDATED(attachment: String) = "Attachment state updated: $attachment"
+    fun MESSAGE_HISTORY_UPDATED(thisMessage: String) = "Message history updated with: $thisMessage"
+    const val CONNECT = "connect"
+    const val CONNECT_AUTHENTICATED_SESSION = "connectAuthenticatedSession"
+    const val DISCONNECT = "disconnect"
+    fun CONFIGURE_AUTHENTICATED_SESSION(token: String, startNew: Boolean) =
+        "configureAuthenticatedSession(token = $token, startNew: $startNew)"
+    fun CONFIGURE_SESSION(token: String, startNew: Boolean) =
+        "configureSession (token = $token, startNew: $startNew)"
+    fun SEND_MESSAGE(text: String, customAttributes: String) =
```

Comment:
SEND_MESSAGE is duplicated. It is definitely going to cause some confusion. 
I suggest to leave the one with custom attributes and reuse it.